libtelemetry/src/main/java/com/mapbox/android/telemetry/NavigationMetadata.java

```diff
@@ -320,6 +365,14 @@ private NavigationMetadata(Parcel in) {
     batteryPluggedIn = in.readByte() != 0x00;
     batteryLevel = in.readInt();
     connectivity = in.readString();
+    tripIdentifier = in.readString();
+    legIndex = in.readInt();
```

Comment:
i assume readInt() returns primitive, would this assignment have a performance implication?

Reply:
see above